addons/mail/static/src/core/web/chatter.js

```diff
@@ -199,16 +202,24 @@ export class Chatter extends Component {
      */
     get toFollowersText() {
         const threadName = this.state.thread.displayName || this.state.thread.name;
+        const followers = [...this.state.thread.followers]
+            .reverse()
+            .slice(-5)
+            .map(({ partner }) =>
+                sprintf('<span title="%(email)s">%(emailWithoutDomain)s</span>', {
+                    email: escape(partner.email),
+                    emailWithoutDomain: escape(partner.emailWithoutDomain),
+                })
+            );
+        const formatter = new Intl.ListFormat(
+            this.store.env.services["user"].lang?.replace("_", "-"),
+            { type: "conjunction" }
+        );
         const toFollowersText = threadName
-            ? sprintf(
-                  _t(
-                      '<span class="fw-bold">To:</span> <span class="fst-italic">Followers of</span> <span class="fw-bold">"%(thread name)s"</span>.'
-                  ),
-                  { "thread name": escape(threadName) }
-              )
-            : _t(
-                  '<span class="fw-bold">To:</span> <span class="fst-italic">Followers of</span> this document.'
-              );
+            ? sprintf("%(followers)s.", {
+                  followers: formatter.format(followers),
+              })
+            : _t('<span class="fst-italic">Followers of</span> this document.');
```

Comment:
I would use `<em>` over `<span class="fst-italic">`.
Reasoning: shorter, better ratio for translated string, and gives room in the future to change style while preserving exact same translated string (less work for translators)

Reply:
This code is not used anymore, I removed it